difflint/lib.py

```diff
@@ -249,7 +197,25 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
+    parser.add_argument('-c', '--check', action='store_true',
+        help='Checks to see if all linting tools are in the PATH. If some ' +
+            'are missing, reports which ones and returns 2.')
+    args = parser.parse_args()
+
+    missing_files = check_linters()
+
+    if args.check:
+        if not missing_files:
+            print('All configuration files and linting programs found.')
+            return 0
+        print(', '.join(missing_files) + ' not found.')
+        return 2
```

Comment:
nit: Would it make sense to use something like https://docs.python.org/3/library/errno.html#errno.EEXIST here?